shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/metadata/dialect/H2DataSourceMetaDataTest.java

```diff
@@ -42,9 +42,31 @@ public void assertNewConstructorWithSymbol() {
         assertThat(actual.getPort(), is(-1));
         assertNull(actual.getSchema());
     }
-    
-    @Test(expected = UnrecognizedDatabaseURLException.class)
```

Comment:
`UnrecognizedDatabaseURLException` might be not used anymore, importing could be removed.